src/Caching/Detector/ChangedFilesDetector.php

```diff
@@ -83,26 +83,14 @@ public function setFirstResolvedConfigFileInfo(string $filePath): void
         $this->storeConfigurationDataHash($filePath, $configHash);
     }
 
-    private function resolvePath(string $filePath): string
```

Comment:
Moved this function to the FileHasher so that we could reuse it in the dependencies cache